translations/base-en.yaml

```diff
@@ -503,7 +503,7 @@ buildings:
     belt:
         default:
             name: &belt Conveyor Belt
-            description: Transports items, hold and drag to place multiple.
+            description: Transports items, click and drag to place multiple.
```

Comment:
Why not change the "hold and drag" on line 1150 as well?
https://github.com/tobspr/shapez.io/blob/fa8857c6242a5a9958f59c4345c982f47613cb5b/translations/base-en.yaml#L1150